src/meta/model/migration/src/m20250820_120000_add_cdc_table_type.rs

```diff
@@ -0,0 +1,36 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
```

Comment:
Is it necessary to refill this column for historical cdc tables?

Reply:
I don't think it's necessary. The history table will be filled with null, which means the toast is not checked(we only check toast when this column is Some(Postgres)). This will not have any impact on the already created CDC table.